test/converter/test_converter.py

```diff
@@ -50,30 +51,42 @@ def test_network_diag_ggn(model_and_input):
     Args:
         model_and_input: module to test
     """
-    model_original, x = model_and_input
+    model_original, x, loss_fn = model_and_input
+    model_original = model_original.eval()
     output_compare = model_original(x)
-    y = rand_like(output_compare)
-
-    num_params = sum(p.numel() for p in model_original.parameters())
+    if isinstance(loss_fn, MSELoss):
+        y = rand_like(output_compare)
```

Comment:
Can you use the `classification_targets` function here?

Reply:
`regression_targets`